src/parser/jobs/rdm/modules/Combos.tsx

```diff
@@ -36,6 +37,22 @@ export class Combos extends CoreCombos {
 	}
 
 	//Overrides
+	override initialise() {
+		super.initialise()
+
+		//Manafication breaks combos, so lets ensure we break when we see it, if we're in a combo!
+		this.addEventHook(
+			filter<Event>()
+				.type('action')
+				.source(this.parser.actor.id)
+				.action(ACTIONS.MANAFICATION.id),
+			this.onTrackedCast)
+	}
+
+	private onTrackedCast(event: Events['action']) {
+		this.onNonDamageCast(event)
```

Comment:
_Consider:_ If you don't wind up needing to put special casing in here to treat starting a spell cast differently from using Manafication, you probably can just pass this.onNonDamageCast in to the addEventHook directly and not need this wrapper (I would expect so anyway, since onNonDamageCast is protected it should be accessible to implementers).

Reply:
Oh good thought, I was doing something up there prior, but I discovered I could simplify what I was doing and removed it.  I'll make that particular change.